WooCommerce/WooCommerceTests/ViewRelated/Products/Product+ProductFormTests.swift

```diff
@@ -18,11 +18,37 @@ class Product_ProductFormTests: XCTestCase {
         let expectedDescription = "This is the party room!"
         XCTAssertEqual(product.trimmedBriefDescription, expectedDescription)
     }
+
+    func testSingleCategoryDescriptionOutputsSingleCategory() {
+        let category = sampleCategory(name: "Pants")
+        let product = sampleProduct(categories: [category])
+        let expectedDescription = "Pants"
+        XCTAssertEqual(product.categoriesDescription, expectedDescription)
+    }
+
+    func testMutipleCategoriesDescriptionOutputsFormattedList() {
+        let categories = ["Pants", "Dress", "Shoes"].map { sampleCategory(name: $0) }
+        let product = sampleProduct(categories: categories)
+        let expectedDescription: String = {
+            if #available(iOS 13.0, *) {
+                return ListFormatter.localizedString(byJoining: ["Pants", "Dress", "Shoes"])
```

Comment:
I'm still not sure how to feel about tests like this. We're simply just repeating the implementation of `.categoriesDescription`. It would probably have been nicer if `ListFormatter.localizedString()` allows us to pass a `locale` so then we could use a full string for comparison instead of having this call here. But it doesn't. ¯\\\_(ツ)\_/¯

Reply:
I wasn't happy with this test either and I turns out that we can pass a locale to format the string.
I've updated the code here https://github.com/woocommerce/woocommerce-ios/pull/2043/commits/44e0467c64ef2365ba658b6262b658ad5cf04cb4